docs/user-guide/leverage-cli/reference/terraform.md

```diff
@@ -159,7 +159,7 @@ leverage terraform validate
 
 Equivalent to `terraform validate`.
 
-Checks the infrastructure definition's consistency. 
+Checks the infrastructure definition's consistency.
```

Comment:
All the other Terraform command's explanations are not using the 3rd person. Just to be consistent, we can take advantage of this change and do one of the following:
- Change "Checks the infrastructure definition's consistency." to "Check the infrastructure definition's consistency." here...
- or change the other commands the other way

Reply:
@juanmatias agree here to keep the consistency, so the 1st bullet sounds like the best choice :)